ros/angel_system_nodes/angel_system_nodes/object_detector_with_descriptors.py

```diff
@@ -334,11 +318,9 @@ def postprocess_detections(self, rois, cls_prob, pooled_feat, im_scales):
             else:
                 objects = torch.argmax(scores[keep_boxes][:,1:], dim=1)
                 box_dets = np.zeros((len(keep_boxes), 4))
-                boxes = pred_boxes[keep_boxes.cpu()]
+                boxes = filled_boxs[keep_boxes].cpu()
```

Comment:
Similarly calling boxes[i].cpu() was the same things as transferring the whole tensor over to cpu earlier